ivy/functional/ivy/activations.py

```diff
@@ -103,6 +129,9 @@ def leaky_relu(
     out
         optional output array, for writing the result to. It must have a shape that the
         inputs broadcast to.
+    complex_mode
```

Comment:
Even though this argument is being handled completely by the wrapper, having this as an argument to the function itself makes things much more intuitive 😄

Reply:
Do you mean I should add it to the parameters inside the `def leaky_relu( ... )` part? I was on the fence about that a little bit, but if its more intuitive with than without then it makes sense to add it. Thanks for the feedback :)

Reply:
Yep, I think that would be more intuitive as the user clearly knows that the functions accepts a `complex_mode`, thanks for making the changes @jshepherd01 😄